src/test/java/spoon/test/imports/ImportScannerTest.java

```diff
@@ -231,94 +228,6 @@ private boolean isTypePresentInStaticImports(String type, Collection<String> sta
 		return imports;
 	}
 
-	@Test
```

Comment:
I guess the whole class should be renamed since `ImportScanner` does not exist anymore

Reply:
Actually I'm not really comfortable with removing those tests: shouldn't they be refactored?

Reply:
you're right the rest if `ImportScannerTest` has been moved to `ImoprtTest`